numpyro/infer/autoguide.py

```diff
@@ -1250,13 +1250,16 @@ def local_model(theta):
         during partial momentum refreshments in HMC. Defaults to 0.9.
     :param float init_scale: Initial scale for the standard deviation of the variational
         distribution for each (unconstrained transformed) local latent variable. Defaults to 0.1.
+    :param str subsample_plate: Optional name of the subsample plate site. This is required
+        when the model does not have subsample plate (like in VAE settings).
+    :param bool use_global_dais_params: Whether to use global parameters for DAIS dynamics.
```

Comment:
nit: Whether parameters controlling DAIS dynamics (HMC step size, HMC mass matrix, etc) should be global (i.e. common to all data points in the subsample plate) or local (i.e. each data point in the subsample plate has individual parameters).

Reply:
done